packages/lib/verification-sender.ts

```diff
@@ -0,0 +1,47 @@
+import { Theme } from 'next-auth';
+import { SendVerificationRequestParams } from 'next-auth/providers';
+import { createTransport, SentMessageInfo, TransportOptions } from 'nodemailer';
+
+import { renderSignInTemplate } from './renderSignInTemplate';
+
+async function sendVerificationRequest(params: SendVerificationRequestParams) {
+  const { identifier, url, provider, theme } = params;
+  const { host } = new URL(url);
+  const transport = createTransport(provider.server);
+  const result: SentMessageInfo = await transport.sendMail({
+    to: identifier,
+    from: provider.from,
+    subject: `Sign in to ${host}`,
+    text: text({ url, host }),
```

Comment:
react-emails has methods for outputing raw text or html, why not using them? 
https://react.email/docs/utilities/render#3-convert-to-html
https://react.email/docs/utilities/render#4-convert-to-plain-text